project/ISSUE-TRIAGE.md

```diff
@@ -31,53 +28,36 @@ If the author provides the standard information but you are still unable to tria
 If the author does not respond requested information within the timespan of a week, close the issue with a kind note stating that the author can request for the issue to be
 reopened when the necessary information is provided.
 
-### Step 2: Apply the template
-
-When triaging, use the standard template below. You should cut and place the template in the issue's description. 
-The template helps other reviewers find key information in an issue. For example, using a template saves a 
-potential contributor from wading though 100s of comments to find a proposed solution at the very end.  When adding 
-the template to the issue's description also add any required labels to the issue for the classification and difficulty.
-
-Here is a sample summary for an [issue](https://github.com/docker/docker/issues/10545).
-
-```
-**Summary**: docker rm can return a non-zero exit code if the container does not
-exist and it is not easy to parse the error message.
-
-**Proposed solution**:
+### 2. Classify the Issue
 
-docker rm should have consistent exit codes for different types of errors so
-that the user can easily script and know the reason why the command failed. 
+An issue can have multiple of the following labels.
 
-```
-
-### Step 3: Classify the Issue
-
-Classifications help both to inform readers about an issue's priority and how to resolve it.
-This is also helpful for identifying new, critical issues.  "Kinds of" are
-applied to the issue or pull request using labels.  You can apply one or more labels.
-
-
-Kinds of classifications:
+#### Issue kind
 
 | Kind             | Description                                                                                                                     |
```

Comment:
- kind/proposal: I hate that one, but I'm fine keeping it ;-)
- kind/helpwanted: meh :-/ what about IRC / ping?
- kind/duplicate, kind/invalid, kind/wontfix: is that really useful considering you'll probably close immediately?
- kind/question: putting this one back per general demand